source/_patterns/basics/10-tags/tag.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tag
+---
+## Overview
+
+This is a Bootstrap component, [see complete documentation here](http://v4-alpha.getbootstrap.com/components/tag/)
+
+NOTE: Bootstrap 4 renames Labels to be Tags in order disambiguate with the <label> tag. See [http://getbootstrap.com/components/#labels](http://getbootstrap.com/components/#labels) for complete labels documentation.
+
+Patternfly design specs are at [http://www.patternfly.org/pattern-library/widgets/#labels](http://www.patternfly.org/pattern-library/widgets/#labels)
+
+Must add a modifier class to add color to the label.
+
+NOTE: Tags contained within `H1`..`H6` are styled slightly differently.
+
+## Accessibility
```

Comment:
We can remove accessibility and usage info from bs components, its already on their docs.